src/jolt/driver/joltcb.py

```diff
@@ -38,7 +38,7 @@
 ID_ASCII = chr(0x4D).encode('latin1')  # packet identifier ascii message
 ID_BIN = chr(0x04).encode('latin1')  # packet identifier binary message
 
-ERROR_CODE = chr(0x59).encode('latin1')  # TODO
+ERROR_CODE = chr(0x8).encode('latin1')  # TODO
```

Comment:
You change it, but is it still "TODO"? If not, drop the comment.

BTW, such code is equivalent to the much simpler:
`ERROR_CODE = b"\x08"`